### DISCUSSION TITLE
Snow vs Maxball

### DISCUSSION THREAD

**Discussion by liuhaogo** [PMEAL/OpenPNM#2326] [Q&A]
Why the number of pores and throats obtained by Snow algorithm and maxball are very different
![_FQVR13~JV59V (2LM99XFE](https://user-images.githubusercontent.com/40398372/154187763-680a14c7-280d-4491-bc3d-f605b123f877.png)
![微信图片_20220216110049](https://user-images.githubusercontent.com/40398372/154187973-824d9e7c-9c1d-4f06-a2d8-9b4c02bb0281.png)

**Comment by jgostick** [comment]
The snow algorithm operates quite differently than the MaxBall.  You can read about the reasons in our PRE paper from 2017.
